agent/host.go

```diff
@@ -107,6 +117,25 @@ func createRootNode(g *graph.Graph) (*graph.Node, error) {
 
 	m.SetField("CPU", cpus)
 
+        // Gets NUMA topology details
+        numaInfo, err := numautils.GetNumaTopology()
```

Comment:
could be nice to add functional tests so that we can ensure that it is working on multiple environment

Reply:
Currently numautils library supports only RHEL, fedora and centos, and also not tested in other env's.
So Planning to add metadata only if no error in function call numautils.GetNumaTopology().
This change also avoids the creating issue in skydive code for unsupported env's. is this fine?

Reply:
@Jaganathancse @safchain Our CI is based on CentOS and Fedora so I don't see any reason why we should not be able to add a test for it in this PR

Reply:
right ! let's test what we can.

Reply:
BTW we need to be sure that it will work with SELinux